### ISSUE TITLE
Update documention for Symfony 2.7

### ISSUE THREAD

**Issue by althaus** [KnpLabs/KnpMenuBundle#269]
Hey,

with the Symfony 2.7 the Definition class will trigger deprecation warnings for the factory construction when using menus as a service.

Before:
```
# app/config/services.yml
services:
    app.sidebar_menu:
        class: Knp\Menu\MenuItem
        factory_service: app.menu_builder
        factory_method: createSidebarMenu
        arguments: ["@request_stack"]
        tags:
            - { name: knp_menu.menu, alias: sidebar } # Named "sidebar" this time
```

After:
```
# app/config/services.yml
services:
    app.sidebar_menu:
        class: Knp\Menu\MenuItem
        factory: ["@app.menu_builder", createSidebarMenu]
        arguments: ["@request_stack"]
        tags:
            - { name: knp_menu.menu, alias: sidebar } # Named "sidebar" this time
```

http://symfony.com/doc/master/bundles/KnpMenuBundle/menu_service.html

Took me a while to find the source of the notices (which were printed plaintext in the page and not collected by the profiler in `dev` environment).

**Comment by weaverryan** [comment]
@althaus about the messages being printed to the screen, have you updated your SensioDistributionBundle? Just wondering - because that should stop that, want to make sure it is :).

This issue itself looks legit - should be a very easy fix.

Cheers!

**Comment by althaus** [comment]
@weaverryan I cannot check it right now, but I did a composer update before and that loaded SensioDistributionBundle as `3.0.something` (found a hint on the KNP page about that issue).

It looked for me like he was printing the warnings when the DependencyInjection stuff required a cache update. I checked that the `bootstrap.php.cache` includes the DEPRECATED stuff. Maybe the issue is related to our quite complex project setup, but I could reproduce it by switching the menu service config back and forth.
